### PR TITLE
[RTL] Fixes for the ManualColumnResize plugin

### PR DESCRIPTION
**Pull request by Siemienik**
### Context

This PR fixes the ManualColumnResize plugin in RTL mode (issue: #8835 )

[skip changelog]

### How has this been tested?

I added new test cases. 

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature or improvement (non-breaking change which adds functionality)

### Related issue(s):
1. #8835 

### Affected project(s):
- [x] `handsontable`

### Checklist:

- [x] I have reviewed the guidelines about [Contributing to Handsontable](https://github.com/handsontable/handsontable/blob/master/CONTRIBUTING.md) and I confirm that my code follows the code style of this project.
- [x] I have signed the [Contributor License Agreement](https://docs.google.com/forms/d/e/1FAIpQLScpMq4swMelvw3-onxC8Jl29m0fVp5hpf7d1yQVklqVjGjWGA/viewform?c=0&w=1)

### CODE REVIEW

**Review by budnix — CHANGES_REQUESTED** [review]
Well done :) However, I found cases where the resizing does not work.

 * The resize handler disappears when I scroll the table horizontally
Kapture 2022-01-18 at 10 06 10

 * The ruler of the handler is placed in the wrong position. It should be placed at the end of the handler.
 LTR mode
image
RTL mode
image

**Comment by Siemienik** [comment]
@budnix I fix this PR, during rereview please keep in mind that the #9134 was reviewed and merged to this PR.

File: handsontable/src/3rdparty/walkontable/src/overlay/_base.js [hunk 1]
```diff
@@ -163,6 +164,18 @@ export class Overlay {
     return offsetObject;
   }
 
+  /**
+   * Get inline start value depending of direction.
+   *
+   * @param {HTMLElement} el Element.
+   * @returns {number|number|number|*}
```
Comment (budnix) [inline handsontable/src/3rdparty/walkontable/src/overlay/_base.js:171]:
```suggestion
   * @returns {number}
```

**Review by budnix — CHANGES_REQUESTED** [review]
I've accidentally found a bug where the handlers are not correctly positioned. It happens when Handsontable is initialized without defining sizing (window is a scrollable element) and on the right overlay.

Kapture 2022-01-26 at 13 38 24

**Comment by Siemienik** [comment]
@budnix would you mind providing me source code to reproduce that?

**Comment by budnix** [comment]
> @budnix would you mind providing me source code to reproduce that?

Sure! https://codesandbox.io/s/vanilla-handsontable-pr-forked-tmvln?file=/src/index.js. If you scroll the page horizontally a little bit the resize handlers do not appear.

**Review by jansiegel — APPROVED** [review]
lgtm 👍

**Review by jansiegel — APPROVED** [review]

**Pull request merged**